plugins/inputs/dcos/README.md

```diff
@@ -0,0 +1,168 @@
+# DC/OS Input Plugin
+
+This input plugin gathers metrics from a DC/OS cluster.
+For more information, please check the [DC/OS Metrics](https://dcos.io/docs/1.9/metrics/) page.
+
+### Configuration:
+
+```toml
+# Input plugin for gathering DC/OS agent metrics
+[[inputs.dcos]]
+  # Hostname or ip address of DC/OS master for access from within DC/OS cluster
+  master_hostname=""
+  # Public URL of DC/OS cluster, e.g. http://dcos.example.com. Use of access from outside of the DC/OS cluster. master_hostname has higher priority, if set
+  #cluster_url=""
+  # Authentication token, obtained by running: dcos config show core.dcos_acs_token. Leave empty for no authentication.
+  # Warning: authentication token is valid only 5 days in DC/OS 1.10.
+  #auth_token=""
+  # List of  DC/OS agent hostnames from which the metrics should be gathered. Leave empty for all.
+  agent_include = []
+  # DC/OS agent node file system mount for which related metrics should be gathered. Leave empty for all.
+  path_include = []
+  # DC/OS agent node network interface names for which related metrics should be gathered. Leave empty for all.
+  interface_include = []
+  # HTTP Response timeout, value must be more than a second
+  #response_timeout = 30s
+  # Set of default allowed tags. See readme.md for more tag keys.
+  taginclude = ["cluster_url","path","interface","hostname","container_id","mesos_id","framework_name"]
```

Comment:
Don't add this to the example config since it is a global filter option.

Reply:
So, you mean to remove it from the readme, but leave it in the plugin default config, right?

Reply:
No, remove it from both places

Reply:
I'm a bit confused. In a previous comment you suggested to filter out some ugly tags. 

Did you mentioned that just for the plugin output example, not as an default option?

Reply:
I can't find what I said about this, maybe you can paste a quote of it?  We shouldn't have the taginclude option in the readme or default config since this is only something the plugin user would use for special filtering.

Reply:
The third comment to readme:
_ executer_name here is very messy, maybe we should whitelist the datapoint tags we want to collect, basically just path and interface _

Reply:
Ahh.  We have a similar situation in the docker input which was setup to add labels as tags.  This worked okay since the user generally sets the labels, until kubernetes starting setting it's own labels which resulted in many useless labels.  Eventually we added the docker_label_include/docker_label_exclude filters specifically for labels.

It seems to me that some of these container tags are not very useful, and so I think we should only collect specific tags that we know are going to be helpful.  In my opinion it is better to add too little than too much.  While the `taginclude` option could be used by the plugin user to further restrict the tags, I'd like to have the plugin behave well without it.

Reply:
Do you mean that the tag white-list should be hardcoded?

Reply:
Yes, it would be fine if they are hardcoded, any tags we add need to be explicit and purposeful.

Reply:
Ok, I will implement it this way: Current tags in taginclude will be filtered in the code, however, if taginclude will be specified by user, hardcoded  filter will be skipped, right?

Reply:
No, have the hardcoded filter be independent of the taginclude, you won't actually have access to it from within the plugin anyway.